tests/src/test/java/org/wildfly/plugins/bootablejar/maven/goals/DevServerTestCase.java

```diff
@@ -41,11 +40,11 @@ public void testDevServer() throws Exception {
         assertEquals("deployment-scanner", mojo.excludedLayers.get(0));
         mojo.execute();
```

Comment:
I think that all the code after this line up to last assert should be placed in try/finally and in the finally shutdown should be called to stop dev server. The last line can remain outside because it calls shutdown anyway.

Reply:
I thought about that as well. What I think I'm going to look at is a `@After` in the `AbstractBootableJarMojoTestCase` because other tests may suffer from leaving a server running as well.

Reply:
#63 should solve this issue.